railties/test/generators/plugin_generator_test.rb

```diff
@@ -244,6 +244,13 @@ def test_no_duplicate_gemfile_entries_when_using_prerelease
     end
   end
 
+  def test_generation_runs_rails_commands
+    generator([destination_root])
+    run_generator_instance
+
+    assert_empty @rails_commands
+  end
```

Comment:
This test also seems unnecessary.  Why would we want to guarantee that the plugin generator doesn't call any `rails` commands?